integrationTests/vm/testIndexer.go

```diff
@@ -168,7 +168,7 @@ func (ti *testIndexer) SaveTransaction(
 	ti.indexer.SaveBlock(blk, header, txsPool, nil, nil, nil)
 
 	// wait --> dataDispatcher needs to save the block
-	time.Sleep(5 * time.Millisecond)
+	time.Sleep(10 * time.Millisecond)
```

Comment:
can we make this time.Second as to be certain that won't fail when the TC agent is experiences high load